src/param.c

```diff
@@ -208,13 +237,27 @@ void param_change_callback(param_id_t id)
   case  PARAM_STREAM_BARO_RATE:
     mavlink_stream_set_rate(MAVLINK_STREAM_ID_BARO, get_param_int(PARAM_STREAM_BARO_RATE));
     break;
+  case  PARAM_STREAM_MAG_RATE:
+    mavlink_stream_set_rate(MAVLINK_STREAM_ID_MAG, get_param_int(PARAM_STREAM_MAG_RATE));
+    break;
 
   case PARAM_STREAM_SERVO_OUTPUT_RAW_RATE:
     mavlink_stream_set_rate(MAVLINK_STREAM_ID_SERVO_OUTPUT_RAW, get_param_int(PARAM_STREAM_SERVO_OUTPUT_RAW_RATE));
     break;
   case PARAM_STREAM_RC_RAW_RATE:
     mavlink_stream_set_rate(MAVLINK_STREAM_ID_RC_RAW, get_param_int(PARAM_STREAM_RC_RAW_RATE));
     break;
+
+  case PARAM_RC_TYPE:
```

Comment:
I did this on the CXX branch with std::bind and std::function.  It's pretty sweet!